### ISSUE TITLE
no extension method 'addUnitOfWork'

### ISSUE THREAD

**Issue by gregorypilar** [Arch/UnitOfWork#11]
Hi,

I just installed the package  and I'm getting this error:

` 'IServiceCollection' does not contain a definition for 'addUnitOfWork' and no extension method 'addUnitOfWork' accepting a first argument of type 'IServiceCollection' could be found (are you missing a using directive or an assembly `

with this code

```
 services.AddDbContext<RegistrosDataModel>(options => options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")))
                .addUnitOfWork<RegistrosDataModel>();
```

this are all the namespaces imported

> using Microsoft.AspNetCore.Builder;
> using Microsoft.AspNetCore.Hosting;
> using Microsoft.AspNetCore.SpaServices.Webpack;
> using Microsoft.Extensions.Configuration;
> using Microsoft.Extensions.DependencyInjection;
> using Microsoft.Extensions.Logging;
> using Microsoft.EntityFrameworkCore;
> using Registros.Model;
> using Newtonsoft.Json.Serialization;
> using Newtonsoft.Json;
>

**Comment by xyting** [comment]
This extension method haven't publish now, I package this today later.

**Comment by xyting** [comment]
@gregorypilar The AddUnitOfWork extension method had been published, please install [v1.1.2](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.UnitOfWork/1.1.2)
